modules/caddyhttp/reverseproxy/caddyfile.go

```diff
@@ -155,6 +155,14 @@ func (h *Handler) UnmarshalCaddyfile(d *caddyfile.Dispenser) error {
 		// the underlying JSON does not yet support different
 		// transports (protocols or schemes) to each backend,
 		// so we remember the last one we see and compare them
+		if pa.scheme == "wss" {
+			return d.Errf("Please use https instead of wss at %s", address)
+		} else if pa.scheme == "ws" {
+			return d.Errf("Please use http instead of ws at %s", address)
+		} else if pa.scheme != "" && pa.scheme != "https" && pa.scheme != "http" && pa.scheme != "h2c" {
+			return d.Errf("Unsupported URL Scheme/Prefix at %s", address)
+		}
+
```

Comment:
At this point, it's probably best/cleaner to use a `switch`.

The error messages typically start with lowercase letters.

Reply:
Thanks for the amazing feedback @mohammed90. Let me take these points into consideration and refactor the code.